src/ServiceControl/ExternalIntegrations/RemoveVersionInformationBehavior.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace ServiceControl.ExternalIntegrations
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.Pipeline;
+
+    public class RemoveVersionInformationBehavior : Behavior<IOutgoingPhysicalMessageContext>
+    {
+        public override Task Invoke(IOutgoingPhysicalMessageContext context, Func<Task> next)
+        {
+            if (context.Headers.TryGetValue(EnclosedTypesHeaderName, out var qualifiedTypeName)
```

Comment:
Not that this is likely to change any time soon, but there's an NSB constant for this: `Headers.EnclosedMessageTypes`